### ISSUE TITLE
Loss value reaches inf in the first epoch

### ISSUE THREAD

**Issue by PrashastiSachan** [Mr-TalhaIlyas/TSFD-Net-for-Nuclei-Segmentation-and-Classification#6]
Hi

Thanks for sharing your work. I am running your code on TensorFlow version 2.6 but I get inf error. Looks like the loss value is too large and thereby I get this error. 

Can you please let me know what's going wrong.

![image](https://user-images.githubusercontent.com/35075466/181920302-4b172325-581b-4a31-9d43-00b299cf0f0d.png)

**Comment by Mr-TalhaIlyas** [comment]
HI,
If you are using multi gpus (as model is quite heavy) then tf and keras does create some problem, we also faced that problem during initial setup. You could follow the solutions listed [#36224](https://github.com/tensorflow/tensorflow/issues/36224) or [here](https://discuss.tensorflow.org/t/getting-nan-for-loss/4826)

**Comment by Mr-TalhaIlyas** [comment]
Also you could check the if you are using the correct combination of loss functions `losses.py` script.
